api/v1alpha4/virtualmachinegroup_publishrequest_types.go

```diff
@@ -0,0 +1,196 @@
+// © Broadcom. All Rights Reserved.
+// The term “Broadcom” refers to Broadcom Inc. and/or its subsidiaries.
```

Comment:
Nit - I thought we agreed to use the standard quote `"` in all the copyright header?

Reply:
This PR predates that, but yes, I'll change it.